indico/web/templates/forms/_access_list.html

```diff
@@ -0,0 +1,25 @@
+<div class="i-box-group vert">
+    {% if users or groups %}
+        <div class="i-box just-group-list">
+            <div class="i-box-content">
+                <ul class="group-list">
+                    {% for user in users %}
+                        <li class="icon-user">{{ user.full_name }}</li>
+                    {% endfor %}
+                    {% for group in groups %}
+                        <li class="icon-users">{{ group.name }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+        </div>
+    {% endif %}
+    <div class="action-box message-only for-form highlight" style="display: block;">
+        <div class="section">
+            <div class="text">
+                <div>
+                    {% trans %}Users with modification access over this resource and any parent resources will also have access{% endtrans %}.
```

Comment:
Thanks for the very useful comments!
As for making the dialog more fancy, I would first consult @pferreir but I would suggest to leave it for later since we're under pressure now.